.github/workflows/ci.yaml

```diff
@@ -11,15 +11,15 @@ jobs:
       VUFIND_LOCAL_DIR: $GITHUB_WORKSPACE/local
     strategy:
       matrix:
-        php-version: ['8.0', '8.1', '8.2']
+        php-version: ['8.1', '8.2', '8.3']
         # We run most tests on all platforms, but we only run Javascript-related tests in 8.1.
```

Comment:
> we only run Javascript-related tests in **8.1**

8.2 now?

Reply:
Good catch. I've totally rewritten the comment to be more detailed and future-proof.